securedrop/journalist_templates/config.html

```diff
@@ -32,6 +32,9 @@ <h2>{{ gettext('Logo Image') }}</h2>
     {{ form.logo(id="logo-upload") }}
     <br>
   </p>
+  <h5>
+    {{gettext('Recommended size: 500px * 450px')}}
```

Comment:
Two things here:

1. We use `{{ gettext(...) }}` (with spaces) stylistically to improve readability.
2. Is there a reason to suggest images that are 500x450 pixels but we thumbnail to 500x500 pixels max?

Reply:
1. Thanks for letting me know. I'll adjust that.
2. `Pillow` suggests giving size as a 2-tuple. The `thumbnail` method automatically adjusts the lower dimension according to aspect ratio. 500 is the upper limit.

If a user uploads square image with edge length greater than 500px, the end result will be 500px by 500px image, since aspect ratio is preserved. It will be resized to 500px by 450px only when the initial aspect ratio was 10:9.

I think the size 500px by 450px comes from the size of default SecureDrop logo. Maybe we can recommend a square image like 500px by 500px.

Reply:
I'm fine with either, but what we suggest and what we thumbnail to should be the same number just to avoid unexpected behavior.

Reply:
I changed the maximum possible height of thumbnail to 450px. If defualt logo can be created in 500*500, we can start recommending 500*500